docs/source/science/GEDI/GEDI_L4A.ipynb

```diff
@@ -2,11 +2,57 @@
  "cells": [
```

Comment:
<p>I think Folium can be capitalized in the title? The docs for it seem to capitalize it in certain areas</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>